src/avram/save_operation_template.cr

```diff
@@ -31,6 +31,8 @@ class Avram::SaveOperationTemplate
         :{{ primary_key_name.id }}
       end
 
+      private def validates_polymorphic_association; end
```

Comment:
This can be removed if using the suggestion about using `before_save` with a block